src/components/CategoryCard.jsx

```diff
@@ -0,0 +1,24 @@
+import { Link } from "react-router-dom";
+
+const CategoryCard = ({ categories }) => {
```

Comment:
Card's are a UI term that refers to an individual element that looks like a physical playing card. The `li`'s in here could be considered cards but not the whole component, this should really get re-named.